trin-utils/src/db.rs

```diff
@@ -0,0 +1,236 @@
+use std::os::unix::fs::symlink;
+use std::path::PathBuf;
+use std::{env, fs};
+
+use anyhow::{anyhow, bail};
+use directories::ProjectDirs;
+use discv5::enr::{CombinedKey, EnrBuilder, NodeId};
+use ethereum_types::H256;
+use tempfile::TempDir;
+use tracing::debug;
+
+use crate::bytes::{hex_decode, hex_encode};
+
+const TRIN_DATA_ENV_VAR: &str = "TRIN_DATA_PATH";
+const TRIN_DATA_DIR: &str = "trin";
+const ACTIVE_NODE_DATA_DIR: &str = "active_node_data_dir.txt";
+const UNSAFE_PRIVATE_KEY_FILE_NAME: &str = "unsafe_private_key.txt";
+
+/// Create a directory on the file system that is deleted once it goes out of scope
+fn setup_temp_dir() -> anyhow::Result<TempDir> {
```

Comment:
No, sorry for the confusion. I never actually witnessed temporary directories that weren't being deleted. Just noticed that we weren't explicitly closing them and assumed that this was something we should be handled. This might have been part of my debugging steps as I was trying to figure out why multiple temp dirs were being created? But, it seems to be unnecessary, as it's all handled by `tempfile` and I've removed it to simplify the code.